api/src/main/webapp/WEB-INF/cron_default.yaml

```diff
@@ -41,24 +61,31 @@ cron:
   target: api
 - description: 'Find and alert users that have exceeded their free tier billing usage'
   url: /v1/cron/checkFreeTierBillingUsage
-  schedule: every 3 hours
+  schedule: every 30 minutes
   timezone: UTC
   target: api
-- description: 'Find billing projects associated with deleted workspaces and transfer ownership to Garbage Collection Service Accounts'
+- description: >
+    Find billing projects associated with deleted workspaces and transfer ownership to
+    designated "Garbage Collection" Service Accounts. This process works around the lack of API
```

Comment:
That would have been true when this API was created. However, this API now exists and we have a ticket to eliminate this cron and move to the new method: https://precisionmedicineinitiative.atlassian.net/browse/RW-3627. I would update this to reflect that and probably link the ticket here